kamon-datadog/src/main/resources/reference.conf

```diff
@@ -37,6 +37,16 @@ kamon {
     # this pattern:
     #    application.entity-name.metric-name
     application-name = "kamon"
+
+    # All time values are collected in nanoseconds,
+    # to scale before sending to datadog set "time-units" to "s" or "ms" or "µs".
+    # Value "ns" is equivalent to omitting the setting
+    # time-units = "ns"
```

Comment:
Why are these values and the ones in the datadog reference file commented out? Shouldn't they be always there?